src/metabase/util/i18n.clj

```diff
@@ -169,10 +169,11 @@
 
 (def ^String ^{:arglists '([& args])} str*
   "Ensures that `trs`/`tru` isn't called prematurely, during compilation."
-  (if *compile-files*
-    (fn [& _]
-      (throw (Exception. "Premature i18n string lookup. Is there a top-level call to `trs` or `tru`?")))
-    str))
+  str
```

Comment:
it was failed to start the jar because of this. temporarily disable it to see if anything else is broken